stories/Modal/Docs.mdx

```diff
@@ -0,0 +1,206 @@
+import { Canvas, Meta } from '@storybook/blocks'
+
+import * as ModalStories from './Modal.stories'
+
+<Meta of={ModalStories} />
+
+# Modal
+
+A modal is a window that can be overlaid on the main window, which when opened, the main window is left partially visible and doesn't allow any user interaction.
+
+## Default
+
+<Canvas of={ModalStories.Default} />
+
+## Attributes
+
+The Modal component is built up of several components, the general structure of which is shown below.
```

Comment:
Are there any attributes that can be passed in to Modal? I would suggest checking out https://www.radix-ui.com/primitives/docs/components/dialog